setup.py

```diff
@@ -19,7 +19,14 @@
 # Converting Pipfile to requirements style list because setup expects requirements.txt file.
 parser = configparser.ConfigParser()
 parser.read("Pipfile")
-install_requires = [f'{key}{value}'.replace('\"', '').replace('*', '') for key, value in parser['packages'].items()]
+
+
+def end_or_comment_index(value: str):
+    return len(value) if value.find("#") == -1 else value.find("#")
+
+
+install_requires = [f'{key}{value[:end_or_comment_index(value)]}'.replace('\'', '').replace('\"', '').replace('*', '')
+                    for key, value in parser['packages'].items() if key != 'demisto-sdk']
```

Comment:
Please add a comment explaining what is going on with the `if key != 'demisto-sdk'`

Reply:
demisto-sdk shouldn't be part of the pipfile at all.

Reply:
@jochman please see my comment

Reply:
I don't think that this is fine. A developer should not install it locally into the same environment he's using. I'm suggesting to remove it.

Reply:
Thats the current recommendation https://github.com/demisto/demisto-sdk/blob/master/CONTRIBUTION.md#2-install-demisto-sdk-dev-environment

Reply:
We advise to install it into the pipenv. Why wouldnt it make sense to install it this way?